openshift/template.yaml

```diff
@@ -36,6 +36,10 @@ objects:
             value: "api"
           - name: F8_API_BACKBONE_HOST
             value: "http://f8a-server-backbone:5000"
+          - name: METRICS_COLLECTOR_HOST
```

Comment:
Openshift will take care of getting these _HOST and _PORT without having an explicit reference in the template as long as that runs in the same project.